src/server/game/Battlegrounds/BattlegroundQueue.cpp

```diff
@@ -963,6 +955,31 @@ void BattlegroundQueue::BattlegroundQueueUpdate(BattlegroundBracketId bracket_id
     }
 }
 
+uint32 BattlegroundQueue::GetPlayesCountInGroupsQueue(BattlegroundBracketId bracketId, BattlegroundQueueGroupTypes bgqueue)
+{
+    uint32 PlayesCount = 0;
```

Comment:
Lower case variable and another Typo in there PlayeR not PlayeS

Reply:
My bad, fix